lib/nimble.phx.gen.template/generator.ex

```diff
@@ -86,4 +111,16 @@ defmodule Nimble.Phx.Gen.Template.Generator do
       [_] -> :error
     end
   end
+
+  defp create_keep_file(path, true) do
+    case File.touch("#{path}/.keep") do
+      :ok ->
+        :ok
+
+      {:error, reason} ->
+        Mix.raise(~s[Failed to create keep file at #{path} reason: #{Atom.to_string(reason)}])
```

Comment:
```suggestion
        Mix.raise(~s[Failed to create .keep file at #{path}. Reason: #{Atom.to_string(reason)}])
```

Reply:
Updated the error message and use comma as a separator.